website/meta/universe.json

```diff
@@ -2463,7 +2463,7 @@
                 "# Load an spacy model (supported models are \"es\" and \"en\") ",
                 "nlp = spacy.load('en')",
```

Comment:
We should probably update this, too?

Reply:
It would be good to update it, but then the code wouldn't work with v2, so I guess we can just leave it for now?

Reply:
Eh, I guess it's pseudocode anyway, I'll add a note.